system/Config/Services.php

```diff
@@ -208,10 +209,12 @@ public static function curlrequest(array $options = [], ?ResponseInterface $resp
 
         $config ??= config(App::class);
         $response ??= new Response($config);
+        $uri = new URI($options['baseURI'] ?? ($options['base_uri'] ?? null));
```

Comment:
parentheses should not needed:

```suggestion
        $uri = new URI($options['baseURI'] ?? $options['base_uri'] ?? null);
```